README.md

```diff
@@ -147,6 +147,7 @@ Some Terraform resources are fully generated, whereas some resources are hand wr
 
 Adding support for a new resource follows these steps:
 
+0. If the resource is already [supported by terraform-google-conversion](https://github.com/GoogleCloudPlatform/terraform-google-conversion/tree/master/google) you can skip to step 6. You may be need to upgrade the terraform-google-conversion dependency in terraform-validator's go.mod file.
```

Comment:
Is this helpful? If not, cool!
```suggestion
0. If the resource is already [supported by terraform-google-conversion](https://github.com/GoogleCloudPlatform/terraform-google-conversion/tree/master/google) you can skip to step 6. You may be need to upgrade the terraform-google-conversion dependency in terraform-validator's [go.mod file](https://github.com/GoogleCloudPlatform/terraform-validator/blob/master/go.mod).
```

Reply:
I was going back and forth on it :-) let's do it!